hw4/04-templates.js

```diff
@@ -12,7 +12,12 @@ app.set('view engine', 'pug');
 const url = 'https://restcountries.com/v3.1/all';
 
 // Add your code here
-
+let arr = [];
+axios.get(url)
```

Comment:
This program seems to be incomplete. Did you forget to push it by any chance? Code seems to be unchanged for  `/populous` and `/regions`

Reply:
I am pushing the changes in /populous and /regions now, thanks for the reminder.